pennylane/templates/subroutines/qsvt.py

```diff
@@ -284,9 +284,32 @@ def __init__(self, UA, projectors, id=None):
         total_wires = ua_wires.union(proj_wires)
         super().__init__(wires=total_wires, id=id)
 
+    @property
+    def data(self):
+        r"""Flattened list of operator data in this QSVT operation.
+
+        This ensures that the backend of a ``QuantumScript`` which contains a
+        ``QSVT`` operation can be inferred with respect to the types of the
+        ``QSVT`` block encoding and projector-controlled phase shift data.
+        """
+        return tuple(datum for op in self._operators for datum in op.data)
+
+    @data.setter
+    def data(self, new_data):
+        if new_data:
```

Comment:
is this check necessary?

Reply:
Yeah, it's necessary because `Operator.__init__()` attempts to assign positional arguments to `self.data` (which is an empty `tuple` in this case). I'll add a comment to explain this.